common/app/com/commercetools/sunrise/myaccount/CustomerFinderBySession.java

```diff
@@ -6,33 +6,32 @@
 import io.sphere.sdk.customers.Customer;
 import io.sphere.sdk.customers.queries.CustomerByIdGet;
 import play.libs.concurrent.HttpExecution;
-import play.mvc.Http;
 
 import javax.inject.Inject;
 import java.util.Optional;
 import java.util.concurrent.CompletionStage;
 
 import static java.util.concurrent.CompletableFuture.completedFuture;
 
-public final class CustomerFinderBySession implements CustomerFinder<Http.Session> {
+public final class CustomerFinderBySession implements CustomerFinder<Void> {
 
     @Inject
     private SphereClient sphereClient;
     @Inject
-    private CustomerSessionHandler customerSessionHandler;
+    private CustomerInSession customerInSession;
     @Inject
     private RequestHookContext hookContext;
 
     @Override
-    public CompletionStage<Optional<Customer>> findCustomer(final Http.Session session) {
-        final CompletionStage<Optional<Customer>> customerStage = fetchCustomer(session);
+    public CompletionStage<Optional<Customer>> findCustomer(final Void unused) {
```

Comment:
This particular implementation of CustomerFinder does not need anything from the developer because it just uses the injected CustomerInSession instance. But that won't always be the case. You could expect the ID, email, external key, orderID... there can be multiple implementations. That's why the parameter is kept there and in the only subclass so far.